pre_award/assess/assessments/templates/assessments/request_changes.html

```diff
@@ -20,15 +22,12 @@
 
 
 {% block content %}
-{{ sub_criteria_heading(sub_criteria)}}
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-one-third">
-        {{ navbar(application_id, sub_criteria, current_theme.id) }}
-    </div>
     <div class="theme govuk-grid-column-two-thirds">
         <div class="govuk-!-margin-bottom-8">
             <h2 class="govuk-heading-l scoring-heading">Reasons for requesting change</h2>
         </div>
+
         {% if form.errors.get("justification") or form.errors.get("field_ids") %}
```

Comment:
Only field_id errors seem to be displayed in this top html element. We only need to check for field_ids.
justification no longer exists. All justification stuff can be removed.